WordPress/gutenberg
# fix: Downgrade Docker image for WordPress

## Description
I've forced our Docker image to be WordPress 4.9.7. If anything, it's possible we want to stick to pinning versions; this sort of upset to our CI/dev environments because of upstream changes is not good.

Fixes #8418.

## How has this been tested?
Run `./bin/setup-local-env.sh`; it worked. Run `npm run test-e2e`; they work.

## Testing instructions

1. Run `./bin/setup-local-env.sh`; it should output the "Gutenberg is ready" text.
2. Run `npm run test-e2e`; the tests should run.
